x-pack/test_serverless/functional/test_suites/observability/infra/hosts_page.ts

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import expect from '@kbn/expect';
+import type { WebElementWrapper } from '../../../../../../test/functional/services/lib/web_element_wrapper';
+import type { FtrProviderContext } from '../../../ftr_provider_context';
+
+import { HOSTS_VIEW_PATH } from './constants';
+import { DATES, DATE_PICKER_FORMAT } from './constants';
+
+const START_DATE = moment.utc(DATES.metricsAndLogs.hosts.min);
+const END_DATE = moment.utc(DATES.metricsAndLogs.hosts.max);
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const browser = getService('browser');
+  const esArchiver = getService('esArchiver');
+  const retry = getService('retry');
+  const testSubjects = getService('testSubjects');
+  const pageObjects = getPageObjects([
+    'assetDetails',
+    'common',
+    'infraHome',
+    'timePicker',
+    'infraHostsView',
+    'settings',
+    'header',
+    'svlCommonPage',
+  ]);
+
+  const waitForPageToLoad = async () =>
+    await retry.waitFor(
+      'wait for table and KPI charts to load',
+      async () =>
+        (await pageObjects.infraHostsView.isHostTableLoading()) &&
+        (await pageObjects.infraHostsView.isKPIChartsLoaded())
+    );
+
+  describe('Hosts Page', function () {
+    before(async () => {
+      await Promise.all([
+        esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs'),
+      ]);
+      await pageObjects.svlCommonPage.login();
+      await browser.setWindowSize(1600, 1200);
+    });
+
+    after(async () => {
+      await Promise.all([
+        esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs'),
+      ]);
+      await pageObjects.svlCommonPage.forceLogout();
+    });
+
+    describe('#Single Host Flyout', () => {
+      before(async () => {
+        // await setHostViewEnabled(true);
```

Comment:
Do we need this line?

Reply:
No, it will be added as part of the settings in [this issue](https://github.com/elastic/kibana/issues/167062) so it won't be needed in this form anymore. Thanks for catching that!